listenstore/listenstore/listenstore.py

```diff
@@ -1,42 +1,36 @@
 # coding=utf-8
-from __future__ import division, absolute_import
+from __future__ import absolute_import
 from __future__ import print_function, unicode_literals
 import logging
 import calendar
-import ujson
```

Comment:
Why are you getting rid of ujson? ujson is considerably faster than the json module. We should not use the json module unless we have to.